cmd/authelia-scripts/cmd_docker.go

```diff
@@ -43,6 +44,15 @@ var DockerPushCmd = &cobra.Command{
 	},
 }
 
+// DockerManifestCmd Command for pushing Authelia docker manifest to Dockerhub
+var DockerManifestCmd = &cobra.Command{
```

Comment:
I think this should be included in the push command.

Reply:
The problem is that the push is executed in each build stage within Travis, as they're all in different hosts.

The manifest needs all of the images to be present before it can be created hence why the deploy step in the current Travis config is only executed if all builds are successful.